osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -858,64 +897,25 @@ - (void)closeSplash {
 
 - (void)showPreferences {
     [splash orderOut:self];
-    [self loadPreferences];
+//    [self loadPreferences];
     [window makeKeyAndOrderFront:self];
     [NSApp activateIgnoringOtherApps:YES];
     //REF http://stackoverflow.com/questions/6994541/cocoa-showing-a-window-on-top-without-giving-it-focus
     [window setLevel:NSFloatingWindowLevel];
 }
 
-
-- (NSString *)getUsernamePref {
-    NSUserDefaults *prefs = [NSUserDefaults standardUserDefaults];
-    NSString *username = [prefs stringForKey:@"username"];
-    return username;
-}
-
-
-- (void)loadPreferences {
-    NSUserDefaults *prefs = [NSUserDefaults standardUserDefaults];
-    self.username = [self getUsernamePref];
-
-    // TODO(cpauya): We must encrypt the password!
-    NSString *password = [prefs stringForKey:@"password"];
-    if (password != nil || [password isNotEqualTo:@""]) {
-        // NSData from the Base64 encoded str
-        NSData *nsdataFromBase64String = [[NSData alloc]
-                                          initWithBase64EncodedString:password options:0];
-        // Decoded NSString from the NSData
-        NSString *decodePassword = [[NSString alloc]
-                                    initWithData:nsdataFromBase64String encoding:NSUTF8StringEncoding];
-        self.password = decodePassword;
-        self.confirmPassword = decodePassword;
-    }
-}
-
-
-- (void)resetPreferences {
-    NSString *blank = @"";
-    self.username = blank;
-    self.password = blank;
-
-    NSUserDefaults *prefs = [NSUserDefaults standardUserDefaults];
-    [prefs setObject:blank forKey:@"username"];
-    [prefs setObject:blank forKey:@"password"];
-    // REF: https://github.com/iwasrobbed/Objective-C-CheatSheet#storing-values
-    [prefs synchronize];
-    [self loadPreferences];
-}
-
+//- (void)loadPreferences {
+    // TODO(amodia): Set custom database path in the preferences menu.
+//}
 
 - (void)savePreferences {
     /*
-     1. Validate the following:
-        * username length max of 30 characters
-        * password length max of 128 characters
-        * username allowed characters are "letters, numbers and @/./+/-/_ characters" based on django.contrib.auth.models.AbstractUser
-     2. Save the preferences: REF: http://stackoverflow.com/questions/10148788/xcode-cocoa-app-preferences
-     3. Run `kalite manage setup` if no database was found.
+     1. Save the preferences: REF: http://stackoverflow.com/questions/10148788/xcode-cocoa-app-preferences
+     2. Run `kalite manage setup` if no database was found.
      */
     
+/*
+    // Comment this to be reference for setting a custom database path in the preferences menu.
```

Comment:
@amodia Pls put TODO(your-github-id-here): in your todos.  I think you can remove the username and password-related codes unless you intend to.